circle.yml

```diff
@@ -25,11 +25,11 @@ test:
     # 0) runs unit tests w/ H2 local DB. Runs against H2, Mongo, MySQL
     # 1) runs unit tests w/ Postgres local DB. Runs against H2, SQL Server
     # 2) runs unit tests w/ MySQL local DB. Runs against H2, Postgres, SQLite
-    # 3) runs Eastwood linter
-    # 4) Bikeshed linter
+    # 3) runs unit tests w/ H2 local DB. Runs against H2, Redshift
```

Comment:
I'm mostly just worried that stuff like this is going to become more common ...

<img width="1056" alt="screen shot 2015-11-30 at 11 08 05 pm" src="https://cloud.githubusercontent.com/assets/1987787/11494450/4de5dd68-97b7-11e5-9d6f-356e69eb84f4.png">

That and ... it's nice to save money, so if the servers are going to sit around for days at a time and never run a single unit test then I'd just assume not have them running.

We aren't in a rush though.  Just something to be thinking about.